<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataPropertiesWidget</class>
 <widget class="QWidget" name="DataPropertiesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>433</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Node Properties</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="KLineEdit" name="_name"/>
   </item>
   <item row="0" column="2">
    <widget class="QCheckBox" name="_showName">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3" rowspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="label_6">
     <property name="toolTip">
      <string>Node color</string>
     </property>
     <property name="text">
      <string>Color:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="KColorCombo" name="_color">
     <property name="toolTip">
      <string>Node color</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Value:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="KLineEdit" name="_value"/>
   </item>
   <item row="1" column="2">
    <widget class="QCheckBox" name="_showValue">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="4" colspan="2">
    <widget class="QCheckBox" name="_disableColor">
     <property name="text">
      <string>Disable Color</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="6">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="6">
    <widget class="QFrame" name="extraItens">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0" rowspan="2" colspan="5">
    <widget class="QTableView" name="_propertiesTable"/>
   </item>
   <item row="5" column="5">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Add Property</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="KLineEdit" name="_propertyName"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Value</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="KLineEdit" name="_propertyValue"/>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QCheckBox" name="_isPropertyGlobal">
        <property name="text">
         <string>Apply to all nodes</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="1">
       <widget class="KPushButton" name="_addProperty">
        <property name="text">
         <string>add</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="5">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>42</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0" colspan="6">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="_btnClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="KComboBox" name="_dataType"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Type:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorCombo</class>
   <extends>QComboBox</extends>
   <header>kcolorcombo.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_name</tabstop>
  <tabstop>_showName</tabstop>
  <tabstop>_propertiesTable</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>_propertyValue</sender>
   <signal>returnPressed()</signal>
   <receiver>_addProperty</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_propertyName</sender>
   <signal>returnPressed()</signal>
   <receiver>_addProperty</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
